portal-api/src/v1/users/sign-in-link.controller.js

```diff
@@ -19,6 +19,8 @@ class SignInLinkController {
 
       const isValidSignInToken = await this.#signInLinkService.isValidSignInToken({ userId: user._id, signInToken });
 
+      await this.#signInLinkService.deleteSignInToken(user._id);
```

Comment:
Shouldn't we only delete it if `isValidSignInToken` is true?

Reply:
Can we update the unit tests for this class with the new logic as well please?

Reply:
`Shouldn't we only delete it if isValidSignInToken is true?`
Yep good point.

`Can we update the unit tests for this class with the new logic as well please?`
Done 👍